tomcat-bin/setenv.sh

```diff
@@ -50,5 +50,12 @@ CATALINA_OPTS="${CATALINA_OPTS} -DSMAAutoNodeDiscovery=true "
 CATALINA_OPTS="${CATALINA_OPTS} -DSMAAutoNodeDiscoveryJMXPort=${JMX_PORT} "
 CATALINA_OPTS="${CATALINA_OPTS} -DSMAAutoNodeDiscoveryPort=8080 "
 
+# Provide setting required for stream node 
+if [ "${NODE_TYPE}" = "Stream" ]; then
```

Comment:
The node_type parameter could contain several different node classifications (such as "WebUser,Backgroundprocessing,Stream".  Can we use the env var IS_STREAM_NODE?